cmd/git-sync/main.go

```diff
@@ -77,15 +76,22 @@ var flMaxSyncFailures = flag.Int("max-sync-failures", envInt("GIT_SYNC_MAX_SYNC_
 var flChmod = flag.Int("change-permissions", envInt("GIT_SYNC_PERMISSIONS", 0),
 	"the file permissions to apply to the checked-out files (0 will not change permissions at all)")
 var flSyncHookCommand = flag.String("sync-hook-command", envString("GIT_SYNC_HOOK_COMMAND", ""),
+	"(deprecated) the command executed with the syncing repository as its working directory after syncing a new hash of the remote repository. "+
+		"it is subject to the sync time out and will extend period between syncs. (doesn't support the command arguments)")
+var flExechookCommand = flag.String("exechook-command", envString("GIT_EXECHOOK_COMMAND", ""),
 	"the command executed with the syncing repository as its working directory after syncing a new hash of the remote repository. "+
```

Comment:
nits, but let's clean up the grammar:

"a command to be executed (without arguments, with the syncing repository as its working directory) after syncing a new hash of the remote repository.  It is subject to --timeout out and will extend period between syncs."

Reply:
updated description.